tests/functional/cylc-install/02-failures.t

```diff
@@ -198,11 +198,10 @@ purge_rnd_workflow
 TEST_NAME="${TEST_NAME_BASE}-nested-rundir"
 make_rnd_workflow
 mkdir -p "${RND_WORKFLOW_RUNDIR}/.service"
-run_fail "${TEST_NAME}-install" cylc install --no-run-name \
-    -C "${RND_WORKFLOW_SOURCE}" \
```

Comment:
I've reverted some of the test changes you made during the course of development as they aren't needed anymore